crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -108,7 +134,7 @@ static void TestOperation(const EVP_CIPHER *cipher, bool encrypt,
 
   bssl::ScopedEVP_CIPHER_CTX ctx;
   // Test running the EVP_Cipher interfaces one by one directly, and check
-  // |EVP_CipherFinal_ex| for approval at the end.
```

Comment:
You also do check `EVP_CipherFinal_ex` as well, correct?

Reply:
See https://github.com/awslabs/aws-lc/blob/af590a7fa59d1d2b6e30df2e4b3aec7dc0c08957/crypto/fipsmodule/service_indicator/service_indicator_test.cc#L150-162. Resolving.